src/com/lftechnology/java/training/sanish/numbergame/NumberGame.java

```diff
@@ -0,0 +1,51 @@
+package com.lftechnology.java.training.sanish.numbergame;
+
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class NumberGame {
+	private static final Logger LOGGER = Logger.getLogger(NumberGame.class
+			.getName());
+
+	public static void main(String[] args) {
+		Scanner scanner = new Scanner(System.in);
+		try {
+			startGuessNumber(scanner);
+		} catch (Exception e) {
+			LOGGER.log(Level.WARNING, "Exception Message : {0}",
+					new Object[] { e.getMessage() });
+		} finally {
+			if (scanner != null)
```

Comment:
you can use try-with-resources feature to close the resources automatically
Have a look at:https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html
http://www.mkyong.com/java/try-with-resources-example-in-jdk-7/